tests/test_helpers.py

```diff
@@ -113,8 +113,16 @@ def test_to_timestamp_none(self):
         # this 'convenience' behaviour has been left left for backwards compatibility
         assert helpers.to_timestamp(None) is None
 
-    def test_from_timestamp(self):
-        assert helpers.from_timestamp(1573431976) == dtm.datetime(2019, 11, 11, 0, 26, 16)
+    def test_from_timestamp_naive(self):
+        datetime = dtm.datetime(2019, 11, 11, 0, 26, 16, tzinfo=dtm.timezone.utc)
```

Comment:
Using `helpers._UTC` here instead of `dtm.timezone.utc` will fix the Python 2 issue—`helpers._UTC`  was made to emulate `dtm.timezone.utc` in Python 2 (in Python 3 it is equivalent to the actual `dtm.timezone.utc`).